README.md

```diff
@@ -53,7 +53,8 @@ itself. They all work out of the box and are completely customizable!
 * Use `paradox-require` instead of `require` to automatically install
   absent packages.
 * Shortcuts for package filtering:
-    * `f r` filters by regexp (`occur`).
+    * `f o` filters by occur.
```

Comment:
You can remove this line. Now that we have proper regexp filtering, there's no need to document occur.